hybris/common/mm/linker.cpp

```diff
@@ -83,9 +83,11 @@ static soinfo* somain; // main process, always the one after libdl_info
 
 static const char* const kDefaultLdPaths[] = {
 #if defined(__LP64__)
+  "/odm/lib64",
```

Comment:
Here /odm is first

Reply:
https://android.googlesource.com/platform/bionic/+/f74b041cce75091b922151d4fa57e63c8a6caf4b%5E!/